### DISCUSSION TITLE
Suggestions / documentation on overriding search logic for autocomplete / select search?

### DISCUSSION THREAD

**Discussion by christopherarter** [livewire/flux#146] [Feature Requests] [answered]
I've got a use case where I need to search with a `LIKE` / `contains` string search so a user can search for "York" and it would pull up the appropriate timezone because it's in the string of the `$value` or `$label`. 

```html
    <flux:select wire:model="selectedTimezone" variant="listbox" searchable placeholder="Choose timezone...">
        <x-slot name="search">
            <flux:select.search placeholder="Search timezone..." />
        </x-slot>
        @foreach($this->timezones as $value => $label)
            <flux:option value="{{ $value }}">{{ $label }}</flux:option>
        @endforeach
    </flux:select>
 ```
 
 
<img width="688" alt="Screenshot 2024-09-26 at 9 52 14 PM" src="https://github.com/user-attachments/assets/7a76c16b-311b-46e4-a31b-d1c4b4818db8">
<img width="674" alt="Screenshot 2024-09-26 at 9 52 21 PM" src="https://github.com/user-attachments/assets/0034bf42-18d3-47c9-9b6c-0018e1435b72">

Any suggestions on if this is possible or how to override? Or is the label the only searchable value? I can just build a custom computed property on a debounce but curious if there's some way this is supported out of the box.

Thanks!

**Comment by patrykszady** [comment]
See https://github.com/livewire/flux/issues/85 and https://github.com/livewire/flux/issues/14 I believe this is what you're after.

**Comment by joshhanley** [comment, marked as answer]
Thanks for reporting! We have converted this issue to a feature request for now, so we can focus on fixing any bugs first.

As there is already a discussion #126 open for this, I'm going to close this discussion, so we can continue the conversation there.
